File: app/src/main/java/com/oec/sdl/vehicle/ShopActivity.java
package com.oec.sdl.vehicle;

import android.os.Build;
import android.os.Bundle;
import android.support.annotation.RequiresApi;
import android.support.design.widget.BottomNavigationView;
import android.support.v7.app.AppCompatActivity;
import android.widget.TextView;

import androidx.navigation.NavController;
import androidx.navigation.Navigation;
import androidx.navigation.ui.AppBarConfiguration;
import androidx.navigation.ui.NavigationUI;

public class ShopActivity extends AppCompatActivity {

    @RequiresApi(api = Build.VERSION_CODES.O)
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_shop);
        // Passing each menu ID as a set of Ids because each
        // menu should be considered as top level destinations.


        ((TextView)findViewById(R.id.textView8)).setText(Integer.toString(SdlService.totalData.getPoint()));
    }

}

File: app/src/main/java/com/oec/sdl/vehicle/ShopActivity.kt
package com.oec.sdl.vehicle

import android.os.Build
import android.os.Bundle
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class ShopActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_shop)
        // Passing each menu ID as a set of Ids because each
        // menu should be considered as top level destinations.


        findViewById<TextView>(R.id.textView8).text = SdlService.totalData.Point.toString()
    }

}
